src/elements/hx-file-icon/index.spec.js

```diff
@@ -0,0 +1,154 @@
+import { fixture, expect } from '@open-wc/testing';
+
+/**
+ * <hx-file-icon> component tests
+ *
+ * @type HXFileIconElement
+ *
+ */
+describe('<hx-file-icon> component tests', () => {
+    const template = '<hx-file-icon>';
+
+    describe('test instantiate element', () => {
+        it('should be instantiated with hx-defined attribute', async () => {
+            const component = /** @type {HXFileIconElement} */ await fixture(template);
+            const attr = component.hasAttribute('hx-defined');
+
+            expect(attr).to.be.true;
+        });
+
+        it('should not be hidden', async () => {
+            const component = /** @type {HXFileIconElement} */ await fixture(template);
+            const prop = component.hidden;
+
+            expect(prop).to.be.false;
+        });
+
+        it(`the rendered Light DOM should NOT equal simple template ${template}`, async () => {
+            const component = /** @type {HXFileIconElement} */ await fixture(template);
+
+            expect(component).lightDom.to.not.equal(template);
+        });
+
+    });
+
+    describe('test Shadow DOM', () => {
+        describe('verify render', () => {
+            it('should have a static Shadow DOM', async function () {
+                const component = /** @type { HXFileIconElement } */ await fixture(template);
+
+                // FIXME: temp fix issue submitted https://github.com/open-wc/open-wc/issues/1540
```

Comment:
Please remove this comment.  It isn't needed.

Reply:
Changes have been done, Please review